### ISSUE TITLE
Accessibility with NVDA on Windows does not work with use case demos

### ISSUE THREAD

**Issue by johannesh2** [vaadin/vaadin-confirm-dialog#29]
The content is read twice with the first run being messed up and the second one being correct.
Additional thing is that the dialog title is read being "clickable", which is a minor thing.

**Comment by johannesh2** [comment]
- [ ] NVDA on Windows
- [x] JAWS on Windows
- [ ] VoiceOver on MaC
